### DISCUSSION TITLE
Mesh Central port

### DISCUSSION THREAD

**Discussion by nicolas5501** [Ylianst/MeshCentral#4477] [Q&A] [answered]
Hello,
I modify on /home/me/meshcentral-data/config.json the ports like that :
    "_port": 5443,
    "_aliasPort": 5443,
    "_redirPort": 80,
    "_redirAliasPort": 5443
But mesh stay on listen on 443. Please, where is my error?
Thanks.

**Comment by si458** [comment, marked as answer]
any variable beginning with an `_ (underscore)` is ignored
so it should be `"port":5443`

**Reply by nicolas5501** [reply]
I'm confuse. So easy. My BIG mistake

Thanks a lot ;)
